coursebuilder/modules/explorer/_static/components/pages/student-dashboard.html

```diff
@@ -0,0 +1,131 @@
+                </div>
+              </div>
+            </div>
+          </template>
+
+          <template is="dom-if" if="[[loaded]]">
+            <template is="dom-if" if="[[!_areCoursesAvailable()]]">
+              <h2>No registered courses.</h2>
+            </template>
+
+            <template is="dom-repeat" items="[[courses]]" as="course">
+
+                <ul class="collapsible" data-collapsible="accordion">
+                  <li>
+                    <div class="collapsible-header">
+                      <p>[[course.title]]</p>
+                    </div>
+                    <div class="collapsible-body">
+
+
+                      <template is="dom-repeat" items="[[course.allUnits.edges]]" as="unit">
+
+                        <div class="row">
+                          <div class="col m12 l6">
+                            <div class="card horizontal">
+                              <div class="card-image">
+                                <img class=sub-one src="images/course/get_online.png">
+                              </div>
+                              <div class="card-stacked">
+                                <div class="card-content">
+                                  <p>[[unit.node.title]]</p>
+                                  <p>
+                                    <template is="dom-repeat"
+                                              items="[[unit.node.detailedLessons.edges]]" as="lesson">
+                                      <template is="dom-if" if="[[lesson.node.progress]]">
+                                        <i class="material-icons teal-text">check_circle</i>
+                                      </template>
+                                      <template is="dom-if" if="[[!lesson.node.progress]]">
+                                        <i class="material-icons teal-text">radio_button_unchecked</i>
+                                      </template>
+                                    </template>
+                                  </p>
+                                </div>
+                                <div class="card-action">
+                                  <a class="start-link" href="#">START</a>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+
+                      </template>
+
+
+                    </div>
+                  </li>
+                </ul>
+
+              </template>
+          </template>
+
+        </div>
+      </div>
+
+    </div>
+  </template>
+
+  <script>
+    Polymer({
+      is: 'student-dashboard',
+      properties: {
+        currentUser: Object,
+        site: Object,
+        loaded: Boolean,
+        courses: Array,
+        coursesAvailable: {
+          type: Boolean,
+          value: false,
+        },
+      },
+      _areCoursesAvailable: function () {
+        if (this.get('courses').length < 1) {
```

Comment:
@andela-angene Are we looking for courses available or courses that a student is currently enrolled in?

Reply:
@chegejames inside the `student-dashboard.html` polymerJs template, `courses` represents the courses that a student is currently enrolled in.

Reply:
@andela-angene a bit confusing, maybe we can have standard naming through out where there is lack of comments.

Reply:
@chegejames The root polymerJs(v1.3) template does not seem to pass properties to child templates when camelCase name is used but I have updated the code to use a more consistent naming so as to avoid confusion.